### DISCUSSION TITLE
In-network-rates schema doesn't allow for plan -> billing code -> provider -> Different negotiated rate per network affiliation

### DISCUSSION THREAD

**Discussion by rakeshkatte** [CMSgov/price-transparency-guide#117] [Q&A] [answered]
There are scenarios where a provider could be affiliated to multiple provider networks, and each network could have negotiated a different rate for a particular service (Billing code). And the plan could have contracted with these networks too. In the current schema, we will end up having two negotiated_rate objects having duplicate information except for the amounts. This could cause confusion to the consumers of the MRF. 
There could be a couple of approaches to take, each having it's own pros & cons.
1. Adding a "negotiation_entity_name" under the in-network object would allow us to report the rates by a provider network name. However, the problem here would be - if the provider has been negotiating the same rate regardless of the network affiliations, then we will have more redundancy in the data.
2. Make the "negotiated_price" object an array and allow for multiple prices along with an identifier for the negotiation_entity_name for each of the prices.
3. There might be other solutions, where the overall structure could lead with a network entity, but that would require a major change to the schema.

Please feel free to let me know if I am missing something and there is a way to represent this data accurately.

**Comment by monty3301** [comment]
Is this a plan with something like a broad and narrow network - same provider contracts with a lower rate in narrow network, more expensive rate in broad network?  Are those different benefit options requiring two sets of IN files?

**Comment by pushpachavan** [comment]
Yes, same plan/provider but different networks, 1 with lower and the other higher. It would still be the one same IN file because the file naming convention only contains the Plan name (and not network).

**Comment by rakeshkatte** [comment]
@monty3301 that could be one of the scenarios, but the size of the network may not be the only driving force there. The benefits (cost share) may not be different at all. Also, if we need to break this into two IN files, we will need to map a bunch of different dynamic "Plan names" to this single plan we have in the system else we will end up with the same name for both the files. So, this might not be a feasible option.

**Comment by hhCambia** [comment]
We have the same concern. @rakeshkatte Option 2 that you outlined eliminates redundancy and seems less disruptive than option 1.

**Comment by ghost** [comment]
@rakeshkatte @hhCambia What percentage of your data to you see this scenario occurring? Wondering because we have not seen the scenario in our data.

**Comment by shaselton-usds** [comment]
@rakeshkatte This is interesting. Moving it to a "discussion".

To make sure I understand, the specific plan (HIOS/EIN id) has multiple provider networks in which a single provider can belong to many of these networks and each network has different negotiated rates for the same items/services? Are these provider networks pretty diverse or the same in terms of providers in them?

How would the consumer (beneficiaries) know what provider network they'd be under if the plan id is the same?

**Reply by rakeshkatte** [reply]
@shaselton-usds 
specific plan (HIOS/EIN id) has multiple provider networks in which a single provider can belong to many of these networks and each network has different negotiated rates for the same items/services? - Yes this understanding is accurate.

Are these provider networks pretty diverse or the same in terms of providers in them? - We have not analyzed the networks to get a measure of how diverse they are, but we just know that this scenario is pretty common.

How would the consumer (beneficiaries) know what provider network they'd be under if the plan id is the same? - The network assignment to a claim is performed during the adjudication process and is done based on various options such as location of the patient, provider or both, or network choice of the member etc., Each plan could adopt one of these options to assign these networks. For ex: Say the patient residence is in Phoenix and plan defines a hierarchy of networks, say N1, N2 and N3, and for Tempe the plan defines the hierarchy as N2, N1 and then N3. Now, if a provider is affiliated to both N1 and N2 and these networks have negotiated different rates, the claim can get processed with different rates depending on where the patient resides.

**Reply by hhCambia** [reply]
We have some groups that offer multiple products to their employees.

For example:

ACME Inc offers two coverages to their employees.
1.	A full PPO network – typically with in and out of network benefits.  
2.	An Exclusive provider network (EPO).   A narrow network.  May be a subset of the PPO but could include providers that are not in the PPO network.

The EPO is meant to drive members to a better value (cost and quality).  Typically there are not any benefits outside of the EPO except for emergent or urgent cases.

In the case above both networks and benefits vary.  

In terms of the 1:N relationship between provider and network, it is common to have a given provider in 2 or more networks across our entire book of business.  It is less common for an EIN/HOIS ID to have more than one network however we are seeing this increasing.

A member that has our coverage understands what network they access. In the case of ACME Inc above, either PPO or EPO.  A member would not have access to both networks and the associated rates. Our IDCards will indicated the networks.

When an EIN has a dual option – PPO and EPO for example, there are two networks under one EIN.  They may or may not have the same providers and when the provider is the same the rate for a service may or may not be the same.

The issue with the current schema is that there is not a way to create multiple networks under the same EIN in the same file.

**Comment by shaselton-usds** [comment, marked as answer]
Wrapping up some old threads -- the [`negotiated_price`](https://github.com/CMSgov/price-transparency-guide/tree/master/schemas/in-network-rates#negotiated-rate-details-object) was turned into an array to avoid duplicated reporting.
